src/Company/Enums.ts

```diff
@@ -0,0 +1,47 @@
+import type { Member } from "../types";
+
+// Imported from Locations and not from the main enums file to avoid circular dependency
+import { LocationName } from "../Locations/Enums";
+
+// CompanyName is essentially a sub-enum of LocationName
+export const CompanyName = {
```

Comment:
why not enum?
I think something like this is possible:

```ts
export enum CompanyName {
  ECorp = LocationName.AevumECorp,
  MegaCorp = LocationName.Sector12MegaCorp,
  BachmanAndAssociates = LocationName.AevumBachmanAndAssociates,
  BladeIndustries = LocationName.Sector12BladeIndustries,
  NWO = LocationName.VolhavenNWO,
  ClarkeIncorporated = LocationName.AevumClarkeIncorporated,
  OmniTekIncorporated = LocationName.VolhavenOmniTekIncorporated,
  FourSigma = LocationName.Sector12FourSigma,
  KuaiGongInternational = LocationName.ChongqingKuaiGongInternational,
  FulcrumTechnologies = LocationName.AevumFulcrumTechnologies,
  StormTechnologies = LocationName.IshimaStormTechnologies,
  DefComm = LocationName.NewTokyoDefComm,
  HeliosLabs = LocationName.VolhavenHeliosLabs,
  VitaLife = LocationName.NewTokyoVitaLife,
  IcarusMicrosystems = LocationName.Sector12IcarusMicrosystems,
  UniversalEnergy = LocationName.Sector12UniversalEnergy,
  GalacticCybersystems = LocationName.AevumGalacticCybersystems,
  AeroCorp = LocationName.AevumAeroCorp,
  OmniaCybersystems = LocationName.VolhavenOmniaCybersystems,
  SolarisSpaceSystems = LocationName.ChongqingSolarisSpaceSystems,
  DeltaOne = LocationName.Sector12DeltaOne,
  GlobalPharmaceuticals = LocationName.NewTokyoGlobalPharmaceuticals,
  NovaMedical = LocationName.IshimaNovaMedical,
  CIA = LocationName.Sector12CIA,
  NSA = LocationName.Sector12NSA,
  WatchdogSecurity = LocationName.AevumWatchdogSecurity,
  LexoCorp = LocationName.VolhavenLexoCorp,
  RhoConstruction = LocationName.AevumRhoConstruction,
  AlphaEnterprises = LocationName.Sector12AlphaEnterprises,
  Police = LocationName.AevumPolice,
  SysCoreSecurities = LocationName.VolhavenSysCoreSecurities,
  CompuTek = LocationName.VolhavenCompuTek,
  NetLinkTechnologies = LocationName.AevumNetLinkTechnologies,
  CarmichaelSecurity = LocationName.Sector12CarmichaelSecurity,
  FoodNStuff = LocationName.Sector12FoodNStuff,
  JoesGuns = LocationName.Sector12JoesGuns,
  OmegaSoftware = LocationName.IshimaOmegaSoftware,
  NoodleBar = LocationName.NewTokyoNoodleBar,
}
```

Reply:
There's technical reasons we're moving away from enums. Snarling might remember the whole rationale.

Reply:
Yeah there was a lot of conversation on "enum bad", but in the end we are sticking with enums and I think they are good (at least the type of enums we usually use, with string values).

I'm not sure why, but I thought we were not able to use computed property values in an enum, but it's just the keys that aren't allowed to be computed values. So yeah I will probably just make it a normal enum. The only advantage of the current method is that ts knows that CompanyNames are also LocationNames, so the company name can also be used as a location name without typechecking/assertions.

Reply:
Changed to enum in 1f224e5